msg/ContactState.msg

```diff
@@ -18,6 +19,15 @@ uint8 manipulation=1
 string name
 uint8 type
 
+# Type of contact states
+int8 UNKNOWN  = -1
+int8 CLOSED   = 0
```

Comment:
Do you mean active and inactive with closed and open?

Reply:
Yes, the contact is closed / active or open / inactive

Reply:
Ok, I think it is better to rename them as ACTIVE and INACTIVE, respectively.
Alternatively, we could add a comment (it wasn't clear for me their meaning).